malaria24/settings/base.py

```diff
@@ -25,7 +26,9 @@
 # See https://docs.djangoproject.com/en/1.7/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = "*y%(_-arf724&tohkw4foyh_4wc6yqe=a7m^wkmz+kw+#pzfx*"
+
+DEFAULT_SECRET_KEY = 'please-change-me'
+SECRET_KEY = environ.get('SECRET_KEY') or DEFAULT_SECRET_KEY + "-"
```

Comment:
This is odd. Why do you need `+ "-"` for?

Reply:
the docker settings check that the secret key and default secret key is not the same, since we don't have the secret key settings in travis yet this was a quick fix to make the tests work

Reply:
I was gonna remove it once the secret key was actually set

Reply:
👍

Reply:
Probably better to set the `SECRET_KEY` envvar for the docker tests. :-)